keras_nlp/models/bert/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 The KerasNLP Authors
```

Comment:
Do we need this file?

Reply:
Well in a strict sense, yes, I think every directory should have an `__init__.py`. But I will make this empty.

We should have some discussion about whether we want a `keras.models.bert` documented namespace. But let's not solve any of that here. Code organization only!